src/community/pom.xml

```diff
@@ -100,6 +100,7 @@
           <descriptor>release/ext-geostyler.xml</descriptor>
           <descriptor>release/ext-csw-iso.xml</descriptor>
           <descriptor>release/ext-importer-jdbc.xml</descriptor>
+          <descriptor>release/ext-jsonld.xml</descriptor>
```

Comment:
Is this file missing? See build failure...

Reply:
fixed